master.tf

```
@@ -59,11 +59,17 @@ resource "hcloud_server" "first_control_plane" {
 
   # Generating k3s master config file
   provisioner "file" {
-    content = templatefile("${path.module}/templates/master_config.yaml.tpl", {
-      node_ip                           = local.first_control_plane_network_ip
-      token                             = random_password.k3s_token.result
-      node_name                         = self.name
-      allow_scheduling_on_control_plane = var.allow_scheduling_on_control_plane
+    content = yamlencode({
+      node-name                = self.name
+      cluster-init             = true
+      disable-cloud-controller = true
+      disable                  = "servicelb, local-storage"
```

I know this works, but I would prefer to use a list here.
```suggestion
      disable                  = ["servicelb", "local-storage"]
```

---

Seems more in line with a YAML list indeed.

---

In the YAML configuration, it is currently:

``` yaml
disable:
- servicelb
- local-storage
```

However, both forms work.

---

Thanks, I'd missed that!